### ISSUE TITLE
Read pdb from list of strings (instead of from file)

### ISSUE THREAD

**Issue by dominiquesydow** [rasbt/biopandas#70]
#### Describe the workflow you want to enable

Thank you for providing this great package - I am using it in most of my projects! 

When a database is queried for structural data in e.g. the pdb format, the file content is often returned in the form of a string.
I would like to load DataFrames from such a string (or list of strings):
https://github.com/volkamerlab/opencadd/blob/912d4e98e89edf38707249fd4f034cea136e1932/opencadd/io/dataframe.py#L128

Currently, I use the private `PandasPdb` method `_construct_df`, which - I know - is bad practice.

#### Describe your proposed solution

In the `mol2` module, we can load DataFrames from a file or from a list of strings.

```python
pmol = PandasMol2()
pmol.read_mol2()
pmol.read_mol2_from_list()
```

Would it be possible to provide the same behavior in the `pdb` module?

```python
ppdb = PandasPdb()
ppdb.read_pdb()
ppdb.read_pdb_from_list()  # New feature?
```

Thank you for your time!

#### Describe alternatives you've considered, if relevant

None.

#### Additional context

None.

**Comment by rasbt** [comment]
Oh yeah, I can totally see how this would be useful. When I developed the library, I was purely going by what I needed for my projects, so I didn't think of it, but I can see that it would be useful (and would make the mol2 vs pdb API more consistent. I am not sure when I get to it (the COVID-19 related teaching workload keeps me super busy this semester), but I will keep that on my todo list for the not so distant future (esp. since this should be implemented rather easily)

**Comment by dominiquesydow** [comment]
@rasbt, thank you for a quick reply despite a full teaching calendar! 

When I started looking into the other issue I posted (#71), I found myself trying to make this issue work, too (see PR #72).
Let me know - when you have time on your hands - if you find this useful.

**Comment by dominiquesydow** [comment]
@rasbt same here closing this issue (addressed in PR #72). Please re-open if necessary.
